Source/PS_ReconPod/Settings/PS_ReconPodSettings.cs

```diff
@@ -16,5 +17,6 @@ public override void ExposeData()
     {
         base.ExposeData();
         Scribe_Values.Look(ref RecondIsBad, "RecondIsBad");
+        Scribe_Values.Look(ref RecondDecays, "RecondDecays");
```

Comment:
should default to enabled, probably (or be inverted). that's what existing users of the mod will expect.